src/main/resources/liquibase/crop_changelog/15_4_1.xml

```diff
@@ -0,0 +1,190 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+				   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+				   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+		http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.5.xsd">
+
+	<changeSet id="v15.4.1-1" author="corina">
+		<preConditions onFail="MARK_RAN">
+			<sqlCheck expectedResult="1">
+				SELECT COUNT(*) > 1 FROM cvtermprop WHERE `value`='Study Detail' AND type_id=1800
+				AND cvterm_id IN (8583, 8584, 8265, 8690, 8700, 8830, 8386, 8720, 8620, 8740, 8750, 8760, 8770,
+									8780, 8387, 8790, 8383, 8194, 8195, 8193, 8191, 8192, 8196, 8800, 8810,
+									8820, 8730, 8384, 8170, 8186);
+			</sqlCheck>
+		</preConditions>
+		<comment> REMOVE Study Detail AS DEFAULT TYPE FOR environment-LEVEL/plot-LEVEL variables. IBP-3889</comment>
+		<sql dbms="mysql" splitStatements="true">
+			SET FOREIGN_KEY_CHECKS=0;
+
+			DELETE FROM `cvtermprop` WHERE `cvtermprop_id` = '8782'; 
```

Comment:
Hi @mchabito ,

   I tried running the checking the cvtermprop records to be deleted in our DBs and I found that one of them had an experimental design variable type (cvtermprop id = [8782](blob:https://ibplatform.atlassian.net/294db4f7-8961-4f8b-a010-d99d49e967f3)). Is this expected?

  My suggestion would be for the delete script is to pattern it after your precondition SQL check:
`DELETE FROM cvtermprop WHERE `value`='Study Detail' AND type_id=1800
				AND cvterm_id IN (8583, 8584, 8265, 8690, 8700, 8830, 8386, 8720, 8620, 8740, 8750, 8760, 8770,8780, 8387, 8790, 8383, 8194, 8195, 8193, 8191, 8192, 8196, 8800, 8810, 8820, 8730, 8384, 8170, 8186);`

Reply:
I agree with the change suggestion